app/json/slate_lineup_configs.json

```diff
@@ -22,7 +22,7 @@
     "experiments": [
       {
         "description": "SlateLineup test 1",
-        "rankers": [ ],
```

Comment:
I regenerated this file to correct the slate order, so select 'Hide whitespace changes' to see only the relevant changes.